canonix/canonix.cabal

```diff
@@ -11,7 +11,7 @@ maintainer:          robert@roberthensing.nl
 -- copyright:
 category:            Language
 extra-source-files:  CHANGELOG.md
-data-files:          tests/golden/*.nix
+                     tests/golden/*.nix
```

Comment:
Commit message should have been: don't install the test suite files.
(at least now it's recorded on github)